sdk/storage/CHANGELOG.md

```diff
@@ -0,0 +1,71 @@
+# Release History
+
+## 1.0.0-preview.1 (Unreleased)
+
+* Added support for Blob features:
```

Comment:
It's not common to use past tense.

Reply:
https://github.com/Azure/azure-storage-ruby/blob/master/ChangeLog.md
https://github.com/Azure/azure-storage-net/blob/master/Blob/Changelog.txt
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/CHANGELOG.md
I think it is pretty common for Azure Storage SDK to use past tense. Can you elaborate why do you think it is not common to use past tense?

Reply:
well i have counter example
https://github.com/microsoft/vcpkg/blob/master/CHANGELOG.md

Reply:
I will leave it as is and can you propose this in next discussion meeting?